switcher.js

```diff
@@ -133,10 +141,29 @@ class Switcher {
     return 'bad';
   }
 
+  /**
+   * Checks that the config file has valid data
+   *
+   * @returns {String} - Status of the file's contents
+   */
+  _checkConfigFile() {
+    if (!fs.existsSync(`${this.awsDir}/config`)) {
```

Comment:
This existence check should really be part of the `_checkDirectory` function, which should also be modified to be something like:

```js
_checkDirectory() {
  if (!fs.existsSync(`${this.awsDir}/config`)) throw new Error(`No config file found in ${this.awsDir}`)
  if (!fs.existsSync(`${this.awsDir}/credentials`)) throw new Error(`No credentials file found in ${this.awsDir}`)
}
```

Then simply called in the constructor replacing lines #26-29

```js
if (!this._checkDirectory(this.awsDir)) {
  throw new Error(`No credentials file found in ${this.awsDir}`);
}
```